core/src/main/java/kafka/server/share/SharePartition.java

```diff
@@ -1338,13 +1340,14 @@ boolean canAcquireRecords() {
      * share partition is not fetched concurrently by multiple clients. The fetch lock is released once
      * the records are fetched and acquired.
      *
+     * @param fetchId - the DelayedShareFetch instance uuid that is trying to acquire the fetch lock.
```

Comment:
Why do we need to talk about `DelayedShareFetch` it's not relevant in SharePartition. It's just uuid which represents the caller's id.